pkg/security/ebpf/c/open.h

```diff
@@ -194,6 +203,14 @@ struct io_open {
     struct openat2_open_how how;
 };
 
+SEC("kprobe/__io_openat_prep")
+int kprobe___io_openat_prep(struct pt_regs *ctx) {
```

Comment:
I'm afraid that open is not the only syscall having the issue. But I think we could add the others later.

Reply:
We should add tests for those then, currently we only test the `openat{,2}` operations through io_uring